### PR TITLE
Make the description of issue templates easier

### PR DESCRIPTION
**Pull request by pklion**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -10,39 +10,37 @@ assignees: ''
...
 🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨
 I ACKNOWLEDGE THE FOLLOWING BEFORE PROCEEDING:
-1. Issue might get deleted if it is not following the template
-2. Only post one issue per bug report
-3. Try to avoid duplicates. Do a quick search before posting
-4. Add a descriptive title
+1. Issue may be deleted if it is not following the template
+2. Only post one issue per one bug report
+3. Try not to make duplicates. Do a quick search before posting
+4. Add a clarified title
```
Comment (yaichenbaum) [inline .github/ISSUE_TEMPLATE/bug_report.md:16]:
```suggestion
4. Add a title that describes the issue clearly
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -10,39 +10,37 @@ assignees: ''
...
-Please upload the log file here so that we can understand your issue better. 
-* You can access it from Settings->About->Open log location. 
+**Logfile**
+Upload the log file here for us to understand your issue better. 
+- You can access it from Settings->About->Open log location. 
 or 
-* `%localappdata%\Packages\49306atecsolution.FilesUWP_et10x9a9vyk8t\LocalState` 
- 
-You should see a file in this directory called debug.txt or/and debug_fulltrust.txt.
+- Copy the debug.txt and debug_fulltrust.txt in the following directory, and paste it here.
```
Comment (yaichenbaum) [inline .github/ISSUE_TEMPLATE/bug_report.md:35]:
```suggestion
- Copy the debug.txt and debug_fulltrust.txt from the following directory, and upload the files here.
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -9,26 +9,22 @@ assignees: ''
...
+1. Issue may be deleted if it is not following the template
+2. Only post one request per one feature request
+3. Try not to make duplicates. Do a quick search before posting
+4. Add a clarified title
 -->
 
-**Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+**Is your feature request related to a problem?d**
+A clear description of what the problem is.d
```
Comment (yaichenbaum) [inline .github/ISSUE_TEMPLATE/feature_request.md:21]:
```suggestion
**Is your feature request related to a problem?**
A clear description of what the problem is.
```

**Comment by pklion** [comment]
@yair100 Thank you!

**Review by yair100 — APPROVED** [review]

**Pull request merged**